diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter5/AsyncAndSyncRuleTest.kt

```diff
@@ -0,0 +1,48 @@
+package org.cqfn.diktat.ruleset.chapter5
+
+import org.cqfn.diktat.ruleset.constants.Warnings.RUN_BLOCKING_INSIDE_ASYNC
+import org.cqfn.diktat.ruleset.rules.AsyncAndSyncRule
+import org.cqfn.diktat.ruleset.rules.DIKTAT_RULE_SET_ID
+import org.cqfn.diktat.util.LintTestBase
+
+import com.pinterest.ktlint.core.LintError
+import generated.WarningNames
+import org.junit.jupiter.api.Tag
+import org.junit.jupiter.api.Test
+
+class AsyncAndSyncRuleTest : LintTestBase(::AsyncAndSyncRule) {
+    private val ruleId = "$DIKTAT_RULE_SET_ID:sync-in-async"
+
+    @Test
+    @Tag(WarningNames.RUN_BLOCKING_INSIDE_ASYNC)
+    fun `test wrong case`() {
```

Comment:
Please add positive example - that `async` and `launch` are allowed to be called from inside `runBlocking`